src/site/stages/build/drupal/api.js

```diff
@@ -248,13 +257,14 @@ function getDrupalClient(buildOptions, clientOptionsArg) {
       // Cap the amount of pending requests allowed out at once
       // And also stagger their execution so that at no point
       // are we totally overwhelming the CMS.
-      const maxParallelRequests = 8;
+      const maxParallelRequests = 15;
       const overallStartTime = moment();
       const staggeredRequests = new Array(maxParallelRequests)
         .fill(null)
-        .map(() => {
+        .map((_, index) => {
           return new Promise(resolve => {
-            setTimeout(() => resolve(parallelQuery()), 1000);
+            const delay = index * 250;
+            setTimeout(() => resolve(parallelQuery()), delay);
```

Comment:
This adds a "stagger" behavior to initial request execution. They are requested with a 250ms delay between each one, which seems better than just blasting the CMS with 15 all at once.